core/src/mender-utils.c

```diff
@@ -509,3 +509,161 @@ mender_utils_append_list(mender_key_value_list_t **list1, mender_key_value_list_
     *list2 = NULL;
     return MENDER_OK;
 }
+
+mender_err_t
+mender_utils_combine_unique(mender_key_value_list_t **list1, mender_key_value_list_t **list2) {
```

Comment:
Below it says the list is appended, why not call the function `append_unique()` then?